interface/ceed-basis.c

```diff
@@ -634,13 +634,21 @@ int CeedSymmetricSchurDecomposition(Ceed ceed, CeedScalar *mat,
       sigma += v[j] * v[j];
     }
     CeedScalar norm = sqrt(v[i]*v[i] + sigma); // norm of v[i:n-1]
-    CeedScalar Rii = -copysign(norm, v[i]);
-    v[i] -= Rii;
+    CeedScalar Rii = copysign(norm, v[i]);
```

Comment:
This `Rii` does not seem to be present in Golub and van Loan's version, but removing it and substituting their use of `norm` into the code below produces results off by more than a sign error. Substituting these changes as is with the use of `Rii` into QR above yields very wrong results, so this can't be correct.

Reply:
I lack context at the moment, but Householder prefers the "further" projection, which I think is the prior code (with the negative).
![TB-Householder2](https://user-images.githubusercontent.com/3303/67358140-aead6780-f51c-11e9-9923-cfafe227d7f8.png)